.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.11.x, 1.16.x]
+        go-version: [1.17.x, 1.19.x]
```

Comment:
Do we not want to continue testing against older versions? Is there a list of Go versions we are supporting?

Reply:
https://cloud.google.com/go/getting-started/supported-go-versions